Docs for redact traces, faster read path, find traces arrays (#7144)
grafana/tempo: docs/sources/tempo/metrics-from-traces/metrics-queries/_index.md

```
{ span:name = "GET /:endpoint" } | quantile_over_time(duration, .99) by (span.http.target) with (exemplars=true)
```

### Faster read path (experimental)

{{< docs/experimental product="Tempo" >}}

> An experimental span-only fetch layer is available in vParquet5 that significantly improves performance for most metrics queries. This optimized read path processes individual spans instead of full traces, reducing latency and memory usage.

Author: ```suggestion
In vParquet5, you can use an experimental span-only fetch layer to significantly improve performance for most metrics queries. This optimized read path processes individual spans instead of full traces, reducing latency and memory usage.
```
